seed/challenges/01-responsive-web-design/css-flexbox.json

```diff
@@ -816,7 +822,7 @@
           "</footer>"
       ],
       "tests": [
-        "assert($('.profile-name').css('justify-content') == 'center' || $('.profile-name').css('justify-content') == 'flex-start' || $('.profile-name').css('justify-content') == 'flex-end' || $('.profile-name').css('justify-content') == 'space-between' || $('.profile-name').css('justify-content') == 'space-around', 'message: The <code>.profile-name</code> element should have the <code>justify-content</code> property set to any of these values: center, flex-start, flex-end, space-between, or space-around.');"
+        "assert(code.match(/header\\s.profile-name\\s*{\\s*?.*?\\s*?.*?\\s*?justify-content\\s*:\\s*(center|flex-start|flex-end|space-between|space-around)\\s*;/g), 'message: The <code>.profile-name</code> element should have the <code>justify-content</code> property set to any of these values: center, flex-start, flex-end, space-between, or space-around.');"
       ],
```

Comment:
I think we need one more set of `\\s*?.*?\\s*?` in the regex before the `justify-content` part, in case the user ignores the blank space for their code and adds the `justify-content: x;` after the `margin-left` property.